service/controller/resource/tccp/create.go

```diff
@@ -105,6 +105,10 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("the tenant cluster's control plane cloud formation stack has stack status %#q", cloudformation.StackStatusCreateInProgress))
 			r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
 			return nil
+		} else if *o.Stacks[0].StackStatus == cloudformation.StackStatusUpdateInProgress {
```

Comment:
I found it super strange we never caught this problem. Now preventing errors where stacks are already in the process of updating.